### ISSUE TITLE
Resize command line parameter not working in Win64 7.0.11-5.

### ISSUE THREAD

**Issue by MatinatorX** [ImageMagick/ImageMagick#3489]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched [open](https://github.com/ImageMagick/ImageMagick/issues) and [closed](https://github.com/ImageMagick/ImageMagick/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported
- [X] I have verified that I am using the latest version of ImageMagick

### ImageMagick version

7.0.11-5 Q16 x64 Windows Standalone, non-HDRI.

### Operating system

Windows

### Operating system, version and so on

Windows 10 Enterprise x64 - v1909

### Description

Resize command not working. 

### Steps to Reproduce

```
convert.exe -resize 25% <infile> <outfile>
convert.exe -resize 25x25  <infile> <outfile>
```
etc.

No errors are given and the command seems to complete properly, but no output image is generated. If the input and output images are the same, the image remains unmodified. Convert.exe --version reports 7.0.11-5 Q16 x64 2021-03-28.

Possibly limited to large images, testing file attached. Verified on multiple machines.
![Test](https://user-images.githubusercontent.com/42086791/113240414-aeb9a700-927a-11eb-90c0-7102090d0fbb.png)

**Comment by fmw42** [comment]
With ImageMagick 7, use magick not convert.  See if that helps

**Comment by snibgo** [comment]
For some years now, IM has expected that operations are listed in the order they are required. Thus: read a file, then resize it, and finally write it to disk.

**Comment by pixel-stuck** [comment]
> For some years now, IM has expected that operations are listed in the order they are required. Thus: read a file, then resize it, and finally write it to disk.

Hi, I'm one of the people who helped realize this was a bug.
`convert <infile> -resize 25% <outfile>` also doesn't output anything.
`convert -scale 25% <infile> <outfile>` does work, and does do the expected operations. So, I just don't buy this.

**Comment by fmw42** [comment]
Disable OpenCL and see if that helps. Also  do try to use the proper syntax, though, for IM 6 is usually does not matter. But for IM 7, it is very important to use the proper syntax.

**Comment by dlemstra** [comment]
This is caused by a mistake that I made when disabling OpenCL by default. We will publish a new release to fix this soon.

**Comment by dlemstra** [comment]
Resolved in release 7.0.11-6.
